specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/customApisDelete.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testResourceGroup",
+    "apiName": "testCustomApi",
+    "api-version": "2016-06-01",
+    "customApi": {
+      "tags": {
+        "sampleTagKey": "sampleTagValue"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {}
+    },
+    "204": {
+      "body": {}
```

Comment:
How do you specify that the body must be empty?

http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html

"The 204 response MUST NOT include a message-body, and thus is always terminated by the first empty line after the header fields."

Reply:
Also could you add a description for each valid response which doesn't have one?

Reply:
Both :-) I noticed in the example files sometimes it is there sometimes it is not, right? But key is for the main swagger file to have it, and for the examples it is about consistency whether it is there or not in the examples.

Reply:
With regards to how to specify empty body response, a) we need to have consistency between our Logic App examples, b) do ask the azure powershell/sdk/rest alias and/or open office hour what is the right way to specify empty.

Reply:
Looking around the rest of the repo, I'm seeing this pattern for delete:

```javascript
  "responses": {
    "200": {},
    "204": {}
  }
```